spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/support/ArgumentSanitizer.java

```diff
@@ -140,10 +146,28 @@ public String sanitizeTask(TaskDefinition taskDefinition) {
 	 */
 	public Map<String, String> sanitizeProperties(Map<String, String> properties) {
 		if (!CollectionUtils.isEmpty(properties)) {
-			return properties.entrySet().stream()
-					.collect(Collectors.toMap(e -> e.getKey(), e -> this.sanitize(e.getKey(), e.getValue())));
+			final Map<String, String> sanitizedProperties = new LinkedHashMap<>(properties.size());
```

Comment:
Any specific reason for this change to move from `stream()` as I don't seem to find the difference.

Reply:
Support for null values :-) See https://stackoverflow.com/questions/24630963/java-8-nullpointerexception-in-collectors-tomap